pages/scenario_selection_page.py

```diff
@@ -191,10 +205,8 @@ def scenario_selection_page():
                                 ashp_loan_interes_rate = st.selectbox(
```

Comment:
```suggestion
                                ashp_loan_interest_rate_option_name = st.selectbox(
```

I think it would be good to differentiate variable names for the interest rate option name and the actual interest rate value for that option